packages/zero-tag-processor/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/zero-tag-processor",
-  "version": "0.0.1-alpha.3",
+  "version": "0.0.1-alpha.4",
```

Comment:
I saw the versions of this package were bumped before so I did the same, but does it really make sense to bump them since they are still private? I think we should put them all back at 0.0.1-alpha.0 and keep them there until the first public release.

Reply:
:+1: BTW this would be consistent with the workflow with pnpm - I omitted the private packages from `lerna version`